corrosion/.github/actions/setup_test/action.yaml

```diff
@@ -138,10 +138,10 @@ runs:
         arch: ${{ steps.arch_flags.outputs.msvc }}
       if: ${{ 'msvc' == steps.determine_abi.outputs.abi }}
     - name: Install CMake
-      uses: corrosion-rs/install-cmake@v2
+      uses: lukka/get-cmake@519de0c7b4812477d74976b2523a9417f552d126
```

Comment:
why this change?

Reply:
These are changes inside the `corrosion` subtree, which I just copied in full from corrosion github